ynr/apps/uk_results/models.py

```diff
@@ -30,16 +36,15 @@ class ResultSet(TimeStampedModel):
     def __str__(self):
         return "Result for {}".format(self.ballot.ballot_paper_id)
 
-    @property
-    def turnout_percentage(self):
+    def calculate_turnout_percentage(self):
         """
         Return turnout as a percentage, rounded to two decimal places
         """
         if not all([self.num_turnout_reported, self.total_electorate]):
             return None
 
         percentage = (self.num_turnout_reported / self.total_electorate) * 100
-        return round(percentage, 2)
+        self.turnout_percentage = round(percentage, 2)
```

Comment:
What happens if this is more than 100? This can happen due to user error of course, but I bet you it can happen due to council error too. Maybe to be save we should wrap this in `min(round(percentage, 2), 100.00)`?